### PR TITLE
Fix Doc Failure on Net Client

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Prevent Doc Generation step from failing on the net - client pipeline

### CODE REVIEW

File: eng/pipelines/templates/steps/archetype-sdk-docs.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ steps:
     workingDirectory: $(Build.BinariesDirectory)
     displayName: Download and Extract Required Software
 
-  - ${{ if ne(variables['Build.DefinitionName'], 'net - client - ci')}}:
+  - ${{ if and(ne(variables['Build.DefinitionName'], 'net - client - ci'), ne(variables['Build.DefinitionName'], 'net - client'))}}:
```
Comment (weshaggard) [inline eng/pipelines/templates/steps/archetype-sdk-docs.yml]: I don't like that we are checking for specific definition names here. We should figure out another way to do this that scales properly. @mitchdenny any thoughts?

Comment (chidozieononiwu) [reply eng/pipelines/templates/steps/archetype-sdk-docs.yml]: Actually if since those pipelines will be deleted, I can just remove this check.

Comment (mitchdenny) [reply eng/pipelines/templates/steps/archetype-sdk-docs.yml]: We are getting rid of the ```net - client - ci``` and ```net - client``` pipelines anyway right?

**Comment by chidozieononiwu** [comment]
No longer relevant since `net - client - ci` and `net - client` has been removed.

**Pull request closed without merging**